packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -530,9 +525,11 @@ function pagePathName(text) {
 }
 
 async function afterTest(compass, test) {
-  if (test.state == 'failed') {
-    await capturePage(compass, screenshotPathName(test.fullTitle()));
-    await savePage(compass, pagePathName(test.fullTitle()));
+  if (process.env.CI) {
```

Comment:
This was inconsistent with how we do it elsewhere. But I think we should decouple this from the CI var and just always store logs and screenshots regardless. Will discuss and do that in a separate PR if we agree on it.

Reply:
I’m good with always doing it.

I know other projects keep a history of screenshots from their CI runs so that it’s easier to identify when some UI/layout bugs were first introduced.